src/Controller/CocktailController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\CocktailFormType;
+use App\Entity\CocktailName;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+
+class CocktailController extends AbstractController
+{
+    #[Route('/home', name: 'home')]
+    public function new(Request $request): Response
+    {
+        $cocktailName = new CocktailName();
+
+        $form = $this->createForm(CocktailFormType::class, $cocktailName);
+        $form->handleRequest($request);
+
+        if($form->isSubmitted() && $form->isValid()){
+
+            $cocktailName = $form->getData();
+
+        }
+
+        return $this->render('cocktail/home.html.twig', [
+            'cocktailForm' => $form->createView(),
```

Comment:
btw which version of symfony are you using? I see createView method is used prior to 5.3 version only.

Reply:
it's prior to 5.3